Makefile

```diff
@@ -347,3 +347,7 @@ KIND_VERSION ?= v0.23.0
 kind: $(KIND) ## Download kind locally if necessary.
 $(KIND): bin/tooling
 	test -s $(KIND) || GOBIN=$(LOCALBIN_TOOLING) $(GO) install sigs.k8s.io/kind@$(KIND_VERSION)
+
+.PHONY: web-build
+web-build: ## Build the web components
+	cd web && npm run build
```

Comment:
is it possible to use `npm --prefix web` to we can run this command from the project root? This would help when chaining multiple make targets together

Reply:
Thanks. I have updated it